src/storage/src/encoding/row_based_deserializer.rs

```diff
@@ -109,6 +156,7 @@ mod tests {
         for (pk, value) in value_bytes {
             assert_eq!(value.len(), 11 + 2 + 3 + 5 + 9 + 5 + 9 + 17 + 17);
             let row1 = de.deserialize(pk, value).unwrap();
+            assert_eq!(DEFAULT_VNODE, row1.clone().unwrap().0);
```

Comment:
Probably can refactor deserialization result to make access to result fields more declarative:
```rs
struct DecodingResult {
    vnode: ...,
    key: ...,
    value: ...,
}
```

Can be done separately too.